module_intros/locality.Filter.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>The second sentence? Also I think it would be better to mention that the strictly-defined blocking condition is something else that can be found in the paper? But this is not super important, just nice to have</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>